Xamarin.Forms.Platform.UAP/RadialGradientBrush.cs

```diff
@@ -0,0 +1,373 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Would need to call out somewhere in a third party notice that you're using this code from the .NET Foundation/Windows Community Toolkit.

Though, you should probably just use the new WinUI 2.4 version as it also supports animations.

Reply:
Do we need to call it out, since we're also part of the .NET Foundation?

Reply:
I think the best way to have RadialGradientBrush is update WinUI to 2.4 https://github.com/xamarin/Xamarin.Forms/pull/10976
Thanks for the feedback @michael-hawker

Reply:
Using WinUI 2.4. 
<img width="912" alt="winui2 4-radial" src="https://user-images.githubusercontent.com/6755973/84127710-e59aa400-aa3f-11ea-85b8-96f15534fd9e.png">